Homework3/tickets.py

```diff
@@ -0,0 +1,31 @@
+"""The new "Avengers" movie has just been released!
+There are a lot of people at the cinema box office standing in a huge line.
+Each of them has a single 100, 50 or 25 dollar bill.
+An "Avengers" ticket costs 25 dollars.
+Vasya is currently working as a clerk.
+He wants to sell a ticket to every single person in this line.
+Can Vasya sell a ticket to every person and give change
+if he initially has no money and sells the tickets strictly
+in the order people queue?
+Return YES, if Vasya can sell a ticket to every person and
+give change with the bills he has at hand at that moment. Otherwise return NO.
+"""
+
+
+def tickets(people):
+    count25, count50 = 0, 0
```

Comment:
this solution looks good but it's not scalable - we can't modify part of it if our people would have 1 5 and 10 dollars bills. we will completely rework this solution and write function that collects change. 
It has the same problem as XO's solution - it's not very flexible and python is about flexibility. 
In general, idea is overall correct and solution is OK for this task.